cli/pcluster/config/param_types.py

```diff
@@ -888,6 +888,15 @@ def create_section_resources(self, section_key, num_labels, max_resources):
         LOGGER.debug("Automatic labels generated: {0}".format(str(section_labels)))
         return self.__section_resources.resources(section_key)
 
+    def to_file(self, config_parser, write_defaults=False):
+        """Ensure cluster_config_metadata_param is not exposed in the config file."""
+        LOGGER.debug("Ensuring cluster_config_metadata_param parameter is not written to configuration file.")
```

Comment:
Here I was following the example set by the Param class. If it's in the config parser, it's going to be written back, right? This would seem to be the safest way to ensure it's never written back.

Reply:
OK, I think it can be good for now.